docs-ref-conceptual/manage-azure-subscriptions-azure-cli.md

```diff
@@ -1,58 +1,161 @@
-To learn how to sign in to a specific tenant, see [Sign in with Azure CLI](./authenticate-azure-cli.md).
+* To learn more about the differences between tenants, users, and subscriptions, see the [Azure cloud terminology dictionary](/azure/azure-glossary-cloud-terminology).
+* To learn how to add a new subscription to your Azure Active Directory tenant, see [Associate or add an Azure subscription to your Azure Active Directory tenant](/azure/active-directory/active-directory-how-subscriptions-associated-directory).
+* To learn how to sign in to a specific tenant, see [Sign in with the Azure CLI](./authenticate-azure-cli.md).
 
 ## Change the active subscription
 
-To access the resources for a subscription, switch your active subscription or use the `--subscription` argument. Switching your subscription
-for all commands is done with [az account set](/cli/azure/account#az-account-set).
+To access the resources for a subscription, switch your active subscription or use the **--subscription** argument. Switching your subscription for all commands is done with [az account set](/cli/azure/account#az-account-set).
 
 To switch your active subscription:
 
 1. Get a list of your subscriptions with the [az account list](/cli/azure/account#az-account-list) command:
 
-    ```azurecli-interactive
+    ```azurecli
     az account list --output table
     ```
-2. Use `az account set` with the subscription ID or name you want to switch to.
 
-    ```azurecli-interactive
+   This command lists all the subscriptions you can access. If you don't see a subscription you expect, add the **--refresh** parameter to get the most current list of subscriptions.
+
+   > [!TIP]
+   > The **--output** parameter is a global parameter, available for all commands. The `table` value presents output in a friendly format. For more information, see [Output formats for Azure CLI commands](/cli/azure/format-output-azure-cli).
+
+1. To see the subscription you're currently using, run the [az account show](/cli/azure/account#az_account_show) command:
+
+   ```azurecli
+   az account show --output table
+   ```
+
+1. To switch to a different subscription, use [az account set](/cli/azure/account#az-account-set) with the subscription ID or name you want to switch to.
+
+    ```azurecli
     az account set --subscription "My Demos"
     ```
 
-To run only a single command with a different subscription, use the `--subscription` argument. This argument takes either a subscription ID or subscription name:
+   Your subscriptions have both a name and an ID, which is a GUID. You can use either for these commands. If you use a name that includes spaces, use quotation marks.
 
-```azurecli-interactive
+To run only a single command with a different subscription, use the **--subscription** argument:
+
+```azurecli
 az vm create --subscription "My Demos" --resource-group MyGroup --name NewVM --image Ubuntu
```

Comment:
If we are trying to show how to use a subscription value other than the default value, please change the subscription demo name to "My second demo" or something.